quantum/color.c

```diff
@@ -30,6 +30,12 @@ RGB hsv_to_rgb( HSV hsv )
 		rgb.r = hsv.v;
 		rgb.g = hsv.v;
 		rgb.b = hsv.v;
+
+#ifdef USE_CIE1931_CURVE
```

Comment:
Why is `rgb` being set twice here?

Reply:
if the linear table is turned of, the fallback needs to still set rgb, though I could use an #else for this instead. Actually, let me do that.

Reply:
fixed